python-avd/pyavd/api/interface_descriptions/__init__.py

```diff
@@ -105,7 +105,9 @@ def underlay_port_channel_interface(self, data: InterfaceDescriptionData) -> str
             - mpls_overlay_role
             - mpls_lsr
             - overlay_routing_protocol
-            - type.
+            - type
+            - wan_carrier
```

Comment:
Do we allow user to configure carrier and circuit id for L3 interfaces as well or only for port channel interfaces?

Reply:
Yes, we allow user to configure these attributes for L3 interfaces as well.
Most of the schema attributes for L3 Port-Channels overlap with L3 Interfaces.